cmd/apps.go

```diff
@@ -50,7 +50,7 @@ func (d DeisCmd) AppCreate(id, buildpack, remote string, noRemote bool) error {
 
 	if !noRemote {
 		if err = git.CreateRemote(s.Client.ControllerURL.Host, remote, app.ID); err != nil {
-			if err.Error() == "exit status 128" {
+			if strings.Contains(err.Error(), fmt.Sprintf("fatal: remote %s already exists.", remote)) {
```

Comment:
#196 is an error that is coming up due to differences in the git versions, but I'm kicking around the idea of using a full git go package to replace our homegrown client, but it's still in the feasibility stage right now.